src/main/external-resources/renderers/Kodi.conf

```diff
@@ -18,7 +18,66 @@ UserAgentSearch = Kodi
 UpnpDetailsSearch = Kodi
 RemoveTagsFromSRTSubtitles = false
 
-# Include all settings from this parent configuration:
-# Note: this must come last for the overrides above to maintain priority
-include = XBMC.conf
+MediaInfo = true
+TranscodeAudio = WAV
+DefaultVBVBufSize = true
+MuxDTSToMpeg = true
+H264Level41Limited = false
+StreamExtensions = mkv,hdmov,hdm,flac,fla,dts,ogg,asf,asx,m2v,avi
+SubtitleHttpHeader = CaptionInfo.sec
+TranscodeFastStart = true
+StreamSubsForTranscodedVideo = true
 
+Supported = f:3gp             v:divx|h263|h264|h265|vc1|mjpeg|mp4|mpeg1|mpeg2|wmv|cvid   a:aac|aac-he|ac3|dts|dtshd|lpcm|mpa|mp2|mp3|wma|truehd        m:video/3gpp
```

Comment:
I haven't paid any attention to the order, I was slowly loosing the will to live as I was trying to test all combinations...

When you say parameters, do you mean the order of the supported lines, to order of the other lines or the "match groups" in the supported lines?